crates/fluvio-extension-common/src/output/table.rs

```diff
@@ -13,6 +12,18 @@ pub trait TableOutputHandler {
     fn errors(&self) -> Vec<String>;
 }
 
+pub trait DisplayTable {
+    fn print_std(&self);
```

Comment:
could you change this function to accept the `indent` and then use that when printing this?